core/src/main/java/io/redlink/smarti/model/Context.java

```diff
@@ -42,6 +42,10 @@
      * conversation
      */
     public static final String ENV_TOKEN = "token";
+    /**
+     * the Channel-Topic
+     */
+    public static final String ENV_SUPPORT_AREA = "support_area";
```

Comment:
Why do you call it support area within the code an channel topic in the comment?
BTW. Earlier we had a discussion about "expertise vs. topics".
A topic has experts and the expert is the one who has expertise.
Within Assistify we have conversations that are not always support cases.
My personal favorite is "topic".

Reply:
Incomplete renaming (good catch :wink:). We first had `expertise` but renamed to `support area`
because in rocket.chat, the field we're using is called `support_area` (see https://github.com/assistify/Rocket.Chat/pull/62)

But it's just a name, if you prefer we can rename again...

Reply:
OK, I also saw that the wording "support_area" has been introduced with the help request feature by @mrsimpson. And you're right it is only naming things. Anyways my preference would be `topic`. @mrsimpson Is that fine with you?